tests/test_measurements.py

```diff
@@ -472,3 +474,14 @@ def test_measurement_basis_list_error(backend):
     c = models.Circuit(4)
     with pytest.raises(ValueError):
         c.add(gates.M(0, 1, 2, 3, basis=[gates.X, gates.Z, gates.X]))
+
+
+def test_CircuitResult_error(backend):
```

Comment:
```suggestion
def test_circuit_result_error(backend):
```
to follow the usual conventions. It's true that we are testing the class but capitals do not look good in function names.